medusa/providers/torrent/html/btdb.py

```diff
@@ -0,0 +1,132 @@
+# coding=utf-8
+
+"""Provider code for BTDB."""
+
+from __future__ import unicode_literals
+
+import logging
+
+from medusa import tv
+from medusa.bs4_parser import BS4Parser
+from medusa.helper.common import (
+    convert_size
+)
+from medusa.logger.adapters.style import BraceAdapter
+from medusa.providers.torrent.torrent_provider import TorrentProvider
+
+from requests.compat import urljoin
+
+log = BraceAdapter(logging.getLogger(__name__))
+log.logger.addHandler(logging.NullHandler())
+
+
+class BTDBProvider(TorrentProvider):
+    """BTDB Torrent provider."""
+
+    def __init__(self):
+        """Initialize the class."""
+        super(BTDBProvider, self).__init__('BTDB')
+
+        # Credentials
+        self.public = True
+
+        # URLs
+        self.url = 'https://btdb.to'
+        self.urls = {
+            'search': urljoin(self.url, '/q/{query}/{page}?sort=popular'),
+        }
+
+        # Cache
+        self.cache = tv.Cache(self, min_time=20)
+
+    def search(self, search_strings, age=0, ep_obj=None, **kwargs):
+        """
+        Search a provider and parse the results.
+
+        :param search_strings: A dict with mode (key) and the search value (value)
+        :param age: Not used
+        :param ep_obj: Not used
+        :returns: A list of search results (structure)
+        """
+        results = []
+
+        for mode in search_strings:
+            log.debug('Search mode: {0}', mode)
+
+            for search_string in search_strings[mode]:
+
+                if mode != 'RSS':
+                    log.debug('Search string: {search}',
+                              {'search': search_string})
+
+                for page in range(1, 4):
+                    search_url = self.urls['search'].format(query=search_string, page=page)
+
```

Comment:
I don't see any code for `Daily search`?

Reply:
What's the difference between daily search and manual search?

Reply:
`Manual search` searches for a specific show (`query=search_string`) and is only executed manually. `Daily search` runs periodically (usually every 30 min) and collects the most recently added elements, something like: https://btdb.to/q/x264/?sort=time

Reply:
Where I can see that results in Medusa WebUI?
Do you think that makes sense for BTDB? It's not a regular torrent index because user don't upload contents. It's auto discovered from DHT network. IMHO https://btdb.to/q/x264/?sort=time results are almost random...

Reply:
Sorry for the late reply! You can trigger a Daily search with `Manage > Manage Searches` and force Daily Search.